succession.md

```diff
@@ -0,0 +1,41 @@
+# Library Carpentry Advisory Committee Succession Planning
+
+This document outlines processes for recruiting new members to the Library Carpentry Advisory Committee and activities required as part of onboarding and offboarding.
+
+## Process for membership recruitment checklist 
+- Co-Chairs lead the recruitment process
+- Call for current members to indicate if they intend to relinquish roles
+- Identify number of members to replace
+- Draft EOI form [https://docs.google.com/forms/d/11bkkHkQc1PbX5ez7frMtWLGPLWtQWBjmLrU3BGr7k4c/edit](using this template) 
```

Comment:
Might clarify to spell out "Expression of Interest (EOI) form" on this first mention.

Reply:
Thanks for picking that up! I'll submit an update. I havne't figured out how to make changes within the github review process.